test/integration/framework/master_utils.go

```diff
@@ -521,3 +523,10 @@ func FindFreeLocalPort() (int, error) {
 	}
 	return port, nil
 }
+
+// SharedEtcd create an independent etcd server instance in-process.
```

Comment:
The wording seemed contradictory to me;  'shared' and 'independent'.  Perhaps drop independent?

Reply:
right

Reply:
fixed